pkg/strategy/infinity-grid/strategy.go

```diff
@@ -0,0 +1,534 @@
+package infinity_grid
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/service"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/c9s/bbgo/pkg/util"
+	"github.com/sirupsen/logrus"
+)
+
+const ID = "infinity-grid"
+
+var log = logrus.WithField("strategy", ID)
+
+func init() {
+	bbgo.RegisterStrategy(ID, &Strategy{})
+}
+
+// State is the grid snapshot
+type State struct {
+	Orders          []types.SubmitOrder           `json:"orders,omitempty"`
+	FilledBuyGrids  map[fixedpoint.Value]struct{} `json:"filledBuyGrids"`
+	FilledSellGrids map[fixedpoint.Value]struct{} `json:"filledSellGrids"`
+	Position        *types.Position               `json:"position,omitempty"`
```

Comment:
You can check how to use the persistence tag, it's a new feature. 

it's applied in the bollmaker strategy.